TMDbLib/Objects/Exceptions/UnhandledException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMDbLib.Objects.Exceptions
+{
+    public class UnhandledException : TMDbHttpException
```

Comment:
Let's name this "GeneralException".

Reply:
Also. As this is explicitly a protocol issue, we could name this "GeneralHttpException", and include the HttpStatusCode here.